client/sites-dashboard/utils.ts

```diff
@@ -74,6 +74,14 @@ export const isTrialSite = ( site: SiteExcerptNetworkData ) => {
 	return isMigrationTrialSite( site ) || isHostingTrialSite( site ) || isECommerceTrialSite( site );
 };
 
+export const siteDefaultInterface = ( site: SiteExcerptNetworkData ) => {
+	return site?.options?.wpcom_admin_interface;
+};
+
+export const getSiteWpAdminUrl = ( site: SiteExcerptNetworkData ) => {
+	return site?.options?.admin_url;
```

Comment:
What will happen when `admin_url` isn't set? What are the potential scenarios where it isn't set?

Reply:
The cases I can think of might be when a code from a plugin or a theme might somehow wipe the `admin_url` option or remove the value or possibly some custom implementation of the "home url" that renames the option. 

I added a fallback for it in 8007b8b3a1af9f7a1862b675af79c31181a6c06c to go to Calypso if it does not exist.